src/components/Cards/__test__/CommunicationCard.test.js

```diff
@@ -0,0 +1,7 @@
+import renderer from "react-test-renderer";
+import CommunicationCard from "../CommunicationCard";
```

Comment:
the `lint.yml` will throw warnings about relative imports because its set to use `list:strict` and not `lint:fix` ... to avoid this use alias imports 
```javascript
import CommunicationCard from "@/components/Cards/CommunicationCard";
```
Or we can update the github action to use `lint:fix`